phpunit.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit colors="true"
+         bootstrap="tests/bootstrap.php">
+    <testsuites>
+        <testsuite name="phpdwtparser test suite">
+            <directory>tests</directory>
+        </testsuite>
+    </testsuites>
+
+    <logging>
```

Comment:
Unless we have a reason to store these files locally, I remove the logging by default. I use a boilerplate from https://github.com/zendframework/zend-stdlib/blob/master/phpunit.xml.dist